File: Iniciante/PHP/exercise_1041.php
<?php

$valores = readline("Insira as coordenadas: ");

list($eixoX, $eixoY) = sscanf($valores, "%f %f");

switch (true) {
    case ($eixoX > 0 && $eixoY > 0):
        print_r("Q1\n");
        break;
    case ($eixoX < 0 && $eixoY < 0):
        print_r("Q3\n");
        break;
    case ($eixoX > 0 && $eixoY < 0):
        print_r("Q4\n");
        break;
    case ($eixoX < 0 && $eixoY > 0):
        print_r("Q2\n");
        break;
     case ($eixoX !== 0.0 && $eixoY === 0.0):
        print_r("Eixo X\n");
        break;
     case ($eixoX === 0.0 && $eixoY !== 0.0):
        print_r("Eixo Y\n");
        break;
    default:
        print_r("Origem\n");
}

//Concluído!

File: Iniciante/Python/exercises from 1000 to 1099/exercise_1041.py
eixoX, eixoY = [float(x) for x in input().split()]

if eixoX == 0 and eixoY == 0:
    print('Origem')
elif eixoX == 0 and eixoY != 0:
    print('Eixo Y')
elif eixoX != 0 and eixoY == 0:
    print('Eixo X')
else:
    if eixoX > 0:
        if eixoY > 0:
            print('Q1')
        else:
            print('Q4')
    else:
        if eixoY > 0:
            print('Q2')
        else:
            print('Q3')